CONTRIBUTING.md

```diff
@@ -13,3 +15,40 @@ Even if you don't have all of these items covered, please still feel free to sub
 
 Thanks again!
 
+## Code of Conduct
+
+The [Node.js Code of Conduct][] applies to this repo.
+
+[Node.js Code of Conduct]: https://github.com/nodejs/node/blob/master/CODE_OF_CONDUCT.md
+
+## Code Contributions
+
+The nvm project falls under the governance of the post-mortem
+working group which is documented in:
+https://github.com/nodejs/post-mortem/blob/master/GOVERNANCE.md
+
```

Comment:
The link is fine for that group- but that WG is not really relevant for the `nvm` project. I think it would probably just fall to the CTC.